src/Controls/src/Core/Stepper/Stepper.cs

```diff
@@ -46,7 +46,14 @@ public partial class Stepper : View, IElementConfiguration<Stepper>, IStepper
 		//If your increment uses more than 4 significant digits, you're holding it wrong.
 		/// <summary>Bindable property for <see cref="Increment"/>.</summary>
 		public static readonly BindableProperty IncrementProperty = BindableProperty.Create(nameof(Increment), typeof(double), typeof(Stepper), 1.0,
-			propertyChanged: (b, o, n) => { ((Stepper)b).digits = (int)(-Math.Log10((double)n) + 4).Clamp(1, 15); });
+			propertyChanged: (b, o, n) =>
```

Comment:
For this fix can you use a pattern of Appending to the Mapper? 

If you look at `Entry.Mapper.cs` you'll see a version of this.

<img width="661" alt="image" src="https://github.com/user-attachments/assets/3e1e4401-730c-4fc2-bef5-56fb9506fac5">

Reply:
Implemented Appending to Mapper approach for this fix. Could you please review the changes